.gitignore

```diff
@@ -7,12 +7,17 @@
 /static/css/output.css
 
 # Environment variables
-.env*
+.env
+# Allow .env.example to be committed
+# .env.example
```

Comment:
remove the comment here. only allow the example file and no other .env files to avoid leaking creds.